backend/src/main/kotlin/org/genspectrum/dashboardsbackend/model/triggerevaluation/TriggerEvaluator.kt

```diff
@@ -57,6 +59,13 @@ class TriggerEvaluator(
             "${dateField}To" to "$currentDate",
         )
     }
+
+    private fun getLatestVersionFilter(organism: Organism): Map<String, String> {
+        if (organism != Organism.WestNile) {
+            return emptyMap()
+        }
+        return mapOf("versionStatus" to "LATEST_VERSION")
```

Comment:
It should also filter for `isRevocation=false`

Reply:
Thanks for catching that. I had a small voice in my head saying that I am missing something, but I could not find out what.

Reply:
I fixed it.